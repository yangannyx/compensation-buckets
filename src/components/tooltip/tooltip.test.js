import { shallow } from 'enzyme';
import { Tooltip } from './';

describe('Tooltip', () => {
  it('should mount', () => {
    const wrapper = shallow(<Tooltip title="Sample Tooltip" info="yoyoyo" link="someLink" />);
    expect(wrapper).toMatchSnapshot();
  });
});
