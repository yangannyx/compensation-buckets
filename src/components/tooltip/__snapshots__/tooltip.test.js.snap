// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tooltip should mount 1`] = `
<div
  className="container"
>
  <div
    className="anchor"
  >
    ?
  </div>
  <div
    className="popover"
  >
    <div
      className="tooltip-title"
    >
      Sample Tooltip
    </div>
    <div
      className="tooltip-info"
    >
      yoyoyo
    </div>
    <div
      className="tooltip-link"
    >
      <a
        className="link"
        href="someLink"
      >
        Learn More
      </a>
    </div>
  </div>
</div>
`;
